### ISSUE TITLE
egui flex `FlexJustify` options doesn't work

### ISSUE THREAD

**Issue by PPakalns** [lucasmerlin/hello_egui#41]
Testing out egui_flex.

I can not get egui_flex::FlexJustify::SpaceBetween or SpaceAround to work and fully fill one line with leaving large gaps empty.
Wanted to create the following effect: https://css-tricks.com/wp-content/uploads/2018/10/justify-content.svg

```
        egui::TopBottomPanel::top("menu")
            .resizable(false)
            .show(ctx, |ui| {
                ui.set_width(1000.); // Tried to manually give space for egui flex
                egui_flex::Flex::horizontal()
                    .justify(egui_flex::FlexJustify::SpaceAround)
                    .show(ui, |flex| {
                        flex.add(egui_flex::item(), egui::Label::new("test1"));
                        flex.add(egui_flex::item(), egui::Label::new("test2"));
                        flex.add(egui_flex::item(), egui::Label::new("test3"));
                    });
              });
```

From source code it looks like property can be defined, but support is not implemented.

**Comment by lucasmerlin** [comment]
Ah, good catch, it seems I just forgot about those. Do you have a need for them or were you just trying things out? 
Right now you should be able to emulate the effects by adding empty items with flex grow in-between your items

**Comment by PPakalns** [comment]
Thank you, this workaround works really well!

**Comment by lucasmerlin** [comment]
Flex justify has been implemented in #44 😄
